### PR TITLE
STORM-2493: update documents to reflect the changes

### PR DESCRIPTION
**Pull request by vesense**

### CODE REVIEW

File: docs/storm-pmml.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#Storm PMML Bolt
```
Comment (erikdw) [inline docs/storm-pmml.md]: nit: spacing is not consistent here with the other docs that you've added (and the existing ones).  i.e., `#Foo` should be `# Foo`.

Comment (vesense) [inline docs/storm-pmml.md]: Nice catch. Will fix.

Comment (erikdw) [inline docs/storm-pmml.md]: I'm anal like that. ;-)

**Comment by vesense** [comment]
@harshach / @revans2 Can you take a look?

**Pull request merged**